### ISSUE TITLE
Enable build for ppc64le and s390x

### ISSUE THREAD

**Issue by gyliu513** [operator-framework/operator-sdk#2656]
## Feature Request

**Is your feature request related to a problem? Please describe.**
A clear and concise description of what the problem is. Example: "I have an issue when (...)"

**Describe the solution you'd like**
A clear and concise description of what you want to happen. Add any considered drawbacks.

Seems the current build only support x86 and macos, any plan to support build for ppc64le and s390x?

**Comment by gyliu513** [comment]
![image](https://user-images.githubusercontent.com/4461983/76719108-be4f7600-6773-11ea-81c0-3a6056bbde47.png)

I think `openshift-ci-bot` only support x86 now, IBM can provide some machines for POWER and Z for prow cluster.

**Comment by camilamacedo86** [comment]
Shows that the images are already built. See [here](https://travis-ci.org/github/operator-framework/operator-sdk/builds/662563445).

@jmrodri @fabianvf could you please give and hand here?  I understand that for we achieve it we need to start to build and push the images in the release process. Am I right?

**Comment by gyliu513** [comment]
Thanks @camilamacedo86 , Yes, I think we should build and push those images to release process.

So the operator-sdk CI/CD is using both PROW and travis?

**Comment by camilamacedo86** [comment]
Hi, @gyliu513 yes we have the upstream and downstream release.  
The upstream release is doc [here](https://github.com/operator-framework/operator-sdk/blob/master/doc/dev/release.md). Please, feel free to check. 

And then, I think that for we achieve it for upstream we need to call the current makefile targets used in the Travis in the process of the script `./release.sh`. However, it shows done already too. See: https://github.com/operator-framework/operator-sdk/blob/master/Makefile#L107. 

I also can check the `ppc64le` arch in the latest release. See:  https://github.com/operator-framework/operator-sdk/releases

<img width="1046" alt="Screenshot 2020-03-16 at 13 08 53" src="https://user-images.githubusercontent.com/7708031/76761469-61f65180-6787-11ea-8c43-41f3736547c8.png">

Are you not able to use these `ppc64le`?

So, IHMO what we need to check is:
- Are you not able to use these `ppc64le`? if not, why?
- Is the release script generating the `s390x` arch? If yes, then in the next release we just need to upload it. If not, then we would need to fix it. 

c/c @jmrodri @hasbro17 @fabianvf

**Comment by morvencao** [comment]
Looks like the operator-sdk totally use travis to build the artifacts for different platforms.
@camilamacedo86 Is there any plan to replace travis with prow?
We're asking this because prow supports multiple build clusters, so we can easily extend that to support build for other platform(ppc64le and s390x), as described in  https://github.com/operator-framework/operator-sdk/issues/2656#issuecomment-599314301

**Comment by camilamacedo86** [comment]
HI @morvencao, 

> @camilamacedo86 Is there any plan to replace travis with prow?

Note that I am aware of for now at least.  

> We're asking this because prow supports multiple build clusters, so we can easily extend that to support build for other platform(ppc64le and s390x), as described in #2656 (comment)

See that my comment https://github.com/operator-framework/operator-sdk/issues/2656#issuecomment-599528795 shows that it is implemented already.

**Comment by morvencao** [comment]
@camilamacedo86 got that, thanks.

**Comment by gyliu513** [comment]
@morvencao can you help check why kubernetes community deprecate travis but all depend on prow? I think we can share some background here to have some discussion with RH team.

**Comment by joelanford** [comment]
As @camilamacedo86 mentioned, during our next release (today, if all goes to plan), we'll upload official builds of the `operator-sdk` binary to GitHub for:
- x86_64-linux-gnu
- ppc64le-linux-gnu
- s390x-linux-gnu
- x86_64-apple-darwin

Currently, the `master` tags (and `latest` and version tags as of today's planned 0.16 release) in `quay.io/operator-framework` are multi-arch images that support x86_64, s390x and ppc64le for the following base images:
- ansible-operator
- helm-operator
- scorecard-proxy

So to summarize:
- Upstream (this repo), we're using only Travis for CI and image build/push. In our next release, all binaries and base images will be built for x86_64, ppc64le, and s390x
- Downstream (https://github.com/openshift/ocp-release-operator-sdk), we use OpenShift CI, but process for building and distributing binaries and images is still a work in progress.

**Comment by joelanford** [comment]
Since this is merged upstream, I'm going to close this. If anyone feels there is more to be done, we can re-open and discuss more.

I would note that we would be interested in help downstream on these things.
